src/ir.rs

```diff
@@ -198,7 +200,7 @@ pub enum FuncletKind
 	Value,
 	ScheduleExplicit,
 	Inline, // Adopts the constraints of the calling funclet
-	Timeline,
+Timeline,
```

Comment:
Not sure what happened here with the indentation, but it happened a couple times in the file and doesn't seem intentional

Reply:
Definitely not intentional, that's an IDE thing.  Sorry about that, I'll see if I can fiddle with settings?